store/src/java-test/com/zimbra/cs/mailbox/MailboxTestUtil.java

```diff
@@ -85,21 +85,22 @@ public static void initProvisioning() throws Exception {
      * @throws Exception
      */
     public static void initProvisioning(String zimbraServerDir) throws Exception {
-    zimbraServerDir = getZimbraServerDir(zimbraServerDir);
-    System.setProperty("log4j.configuration", "log4j-test.properties");
-    System.setProperty("zimbra.config", zimbraServerDir + "src/java-test/localconfig-test.xml");
-    LC.reload();
-
-    String timezonefilePath = zimbraServerDir + "src/java-test/timezones-test.ics";
-    File d = new File(timezonefilePath);
-    if (!d.exists()) {
-        throw new FileNotFoundException("timezones.ics not found.");
-    }
-    LC.timezone_file.setDefault(timezonefilePath);
-    LC.zimbra_rights_directory.setDefault(StringUtils.removeEnd(zimbraServerDir, "/") +"-conf" + "/conf/rights");
-    LC.zimbra_attrs_directory.setDefault(zimbraServerDir + "conf/attrs");
-    // default MIME handlers are now set up in MockProvisioning constructor
-    Provisioning.setInstance(new MockProvisioning());
+        zimbraServerDir = getZimbraServerDir(zimbraServerDir);
+        System.setProperty("log4j.configuration", "log4j-test.properties");
+        System.setProperty("zimbra.config", zimbraServerDir + "src/java-test/localconfig-test.xml");
+        LC.reload();
+
+        String timezonefilePath = zimbraServerDir + "src/java-test/timezones-test.ics";
+        File d = new File(timezonefilePath);
+        if (!d.exists()) {
+            throw new FileNotFoundException("zm-timezones repository not found. Please clone this repo in same directory as zm-mailbox before running this test.");
```

Comment:
I think this has happened while merging.
We no longer refer to the timezones.ics file in zm-timezones repo, so we should use the following exception message.
```throw new FileNotFoundException("timezones.ics not found.");```